pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -170,13 +170,6 @@
 
 def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by ``lightning.qubit``."""
-    # To avoid building matrices beyond the given thresholds.
-    # This should reduce runtime overheads for larger systems.
-    if isinstance(op, qml.QFT):
-        return len(op.wires) < 10
```

Comment:
Looks like you need to update the testing suite because fails some test :sweat_smile:

Reply:
yeah, working on them :smile: